client/src/index.css

```diff
@@ -1097,3 +1097,24 @@ header.searchPagination {
 		height: 100% !important;
 	}
 }
+
+.merge-loc-fset .legend {
```

Comment:
Could we keep the css defined inline?

Reply:
Here, `.merge-loc-fset` class doesn't define any styles by itself. It primarily exists for selecting dom children in order to override default styling of certain `Fieldset` components in `MergeLocations` page. In order to inline these styles, `Fieldset` component's api needs to change which I think we shouldn't because it seems to me as an overkill for such a change.